README.md

```diff
@@ -19,3 +19,15 @@ The following shortcodes are either ported over from WordPress.com or are create
 The following custom WP-CLI commands exist in this plugin:
 
 * `wpcom-compat import-protected-embeds` - Imports "protected embeds" from a CSV file into the database table `protected_embeds`.
+
+## Convert plain text URLs in `post_content` to links on display.
+
+* [`make_clickable()`](https://core.trac.wordpress.org/browser/tags/4.9.8/src/wp-includes/formatting.php#L2608) is a function in WordPress Core that converts plain text URLs to HTML links.
```

Comment:
I'm not sure if we need this extra bit of context here.  It might be just as useful to link the `make_clickable()` below to the docs at https://developer.wordpress.org/reference/functions/make_clickable/